UKHomeOffice/nmsw-ui
# NMSW-1297 Add download links on FAL5/6 upload pages

# Ticket

NMSW-1297

----

## AC

AC1

GIVEN I am uploading a FAL5
WHEN the upload page is displayed
THEN there is a link to download the FAL5 template

 AC2

GIVEN I am uploading a FAL6
WHEN the upload page is displayed
THEN there is a link to download the FAL6 template

----

## To test

- run app
- create a declaration
- > on General declaration page the wording should be updated
- click on the crew details upload
- > there should be text and a link to download the FAL 5 form
- click on passengers upload
- > there should be text and a link to download the FAL 6 form
----

## Notes
